### PR TITLE
Changed `Utility.InflateRectangles` to return `IEnumerable<T>`

### PR DESCRIPTION
**Pull request by Lehonti**
...However, I don't see it being used anywhere. Is this from a feature that is no longer there?

In any case, I also fixed a few naming violations, made a few formatting changes, and removed unnecessary NRT specifiers

### CODE REVIEW

File: Pinta.Core/Effects/Utility.cs [hunk 1]
```diff
@@ -19,68 +20,62 @@ internal static bool IsNumber (float x)
 
 	public static double Clamp (double x, double min, double max)
 	{
-		if (x < min) {
+		if (x < min)
```
Comment (cameronwhite) [inline Pinta.Core/Effects/Utility.cs:23]: Can these first three `Clamp` methods just be replaced by `System.Math.Clamp()`?

File: Pinta.Core/Effects/Utility.cs [hunk 2]
```diff
@@ -194,14 +188,10 @@ public static int ColorDifferenceSquared (ColorBgra a, ColorBgra b)
 		return diffSq / 3;
 	}
 
-	public static RectangleI[] InflateRectangles (RectangleI[] rects, int len)
+	public static IEnumerable<RectangleI> InflateRectangles (IEnumerable<RectangleI> rects, int len)
```
Comment (cameronwhite) [inline Pinta.Core/Effects/Utility.cs:191]: I'm fine to remove this since it's unused. It was added a long time ago and usages of it were removed since

Comment (Lehonti) [reply Pinta.Core/Effects/Utility.cs]: Nice, the less unused methods the better. I've removed it

File: Pinta.Core/Effects/Utility.cs (hunk 1 above)
Comment (Lehonti) [reply Pinta.Core/Effects/Utility.cs]: Sure. It seems to do the same job. I removed it and replaced all of its occurrences by `System.Math.Clamp()`

Comment (cameronwhite) [reply Pinta.Core/Effects/Utility.cs]: Looks good, thanks!
One further thought is that the ClampToByte methods could be reimplemented as just `(byte) Math.Clamp(x, 0, 255)`?

Comment (Lehonti) [reply Pinta.Core/Effects/Utility.cs]: No problem. Done

**Pull request merged**